index.html

```diff
@@ -0,0 +1,235 @@
+			<div class="container">
+				<div class="news__wrapper">
+					<div class="news__text-column">
+						<img src="pic/Rectangle 3.7.png" width="307px" height="147.24px" alt="Presents Justin Anagoa">
+						<h3 class="news__title">
+							Presents Justin Anagoa
+						</h3>
+						<p>
+							Anagoa’s been a part of the skater’s family ever since his early teenage years. He’s been known to go big on and off the board, but it’s been rare we get a glimpse into how Anagoa lives his day-to-day, to see what makes Anagoa tick.
+						</p>
+						<a href="#" class="learn-more-button">Learn more<img src="pic/Arrow — копия.png" width="28.13px" height="0" alt="arrow"></a>
+					</div>
+					<div class="news__text-column">
+						<img src="pic/Rectangle 3.7-2.png" width="307px" height="147.24px" alt="First Frontside flip">
+						<h3 class="news__title">
+							First Frontside flip
+						</h3>
+						<p>
+							Yoon recently discusses his first time shooting a signature Andrew Reynolds frontside flip on The Berrics. Click the image below to see the full gallery and read up on what happened that day. Photo:Yoon Sul
+						</p>
+						<a href="#" class="learn-more-button">Learn more<img src="pic/Arrow — копия.png" width="28.13px" height="0" alt="arrow"></a>
+					</div>
+					<div class="news__text-column">
+						<img src="pic/Rectangle 3.7-3.png" width="307px" height="147.24px" alt="Kevin Baekkel is Pro">
+						<h3 class="news__title">
+							Kevin Baekkel is Pro
+						</h3>
+						<p>
+							Creature proudly announced today that they’re officially adding Kevin Baekkel to their pro roster. After paying his dues for years, killing it on last years Thrasher King of the Road, and constantly lighting every spot he hits on fire.
+						</p>
+						<a href="#" class="learn-more-button">Learn more<img src="pic/Arrow — копия.png" width="28.13px" height="0" alt="arrow"></a>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="photo-gelery">
+			<div class="container">
+				<div class="photo-gelery__wrapper">
+					<h3 class="photo-gelery__title">
+						#AXIONfootwear
+					</h3>
+					<p class="photo-gelery__pic">
+						<img src="pic/Rectangle 3.png" width="147.14px" height="147.14px" alt="pic 1"><img src="pic/Rectangle 3.1.png" width="147.14px" height="147.14px" alt="pic 2"><img src="pic/Rectangle 3.2.png" width="147.14px" height="147.14px" alt="pic 3"><img src="pic/Rectangle 3.3.png" width="147.14px" height="147.14px" alt="pic 4"><img src="pic/Rectangle 3.4.png" width="147.14px" height="147.14px" alt="pic 5"><img src="pic/Rectangle 3.5.png" width="147.14px" height="147.14px" alt="pic 6"><img src="pic/Rectangle 3.6.png" width="147.14px" height="147.14px" alt="pic 7">
+					</p>
+				</div>
+			</div>
+		</section>
+	</main>
+	<footer class="main-footer">
+		<div class="container">
+			<div class="main-footer__wrapper">
+				<section class="main-footer__column">
+					<h4 class="main-footer__section-title">
+						MY ACCOUNT
```

Comment:
В разметке не нужно вставлять текст в верхнем регистре, это делается стилями
```suggestion
						My account
```